include/m_common_data.h

```diff
@@ -211,7 +212,9 @@ typedef struct CommonData {
     /* 0x10078 */ CommonData_unk_10078 *unk_10078;
     /* 0x1007C */ UNK_TYPE1 unk_1007C[0x1C];
     /* 0x10098 */ CommonData_unk_10098 *unk_10098;
-    /* 0x1009C */ UNK_TYPE1 unk_1009C[0x48];
+    /* 0x1009C */ UNK_TYPE1 unk_1009C[0x4];
+    /* 0x100A0 */ ToolClip* toolClip;
```

Comment:
It may be better to do it this way and avoid the include if we can, to avoid future circular references
```suggestion
    /* 0x100A0 */ struct ToolClip* toolClip;
```